.github/workflows/go.yml

```
@@ -24,3 +24,20 @@ jobs:
       - name: Run non-flaky tests concurrently
         run: |
           go test -short $(go list ./... | grep -Pv "${FLAKY_REGEX}");
+          
+  go_generate:
+    env:
+      EXCLUDE_REGEX: 'ava-labs/libevm/(accounts/usbwallet/trezor)$'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.21.4
+
+      - name: Run `go generate`
+        run: go list ./... | grep -Pv "${EXCLUDE_REGEX}" | xargs go generate;
```

You can use the `-skip` flag I think:

```suggestion
      - run: go generate -skip "${EXCLUDE_REGEX}" ./...
```

---

Thanks, TIL about `-skip`. This doesn't work for me locally though (I suspect because the regex needs to change):

```shell
$ go generate -skip 'ava-labs/libevm/(accounts/usbwallet/trezor)$' ./...
/usr/local/include: warning: directory does not exist.
protoc-gen-go: no such flag -import_path
--go_out: protoc-gen-go: Plugin failed with status code 1.
accounts/usbwallet/trezor/trezor.go:45: running "protoc": exit status 1
```

If you add the [matrix strategy](https://github.com/ava-labs/libevm/pull/86#discussion_r1878460977) and switching to `-skip` is simple then please include it, but I think it should be timeboxed as it's low yield.

---

My apologies, similarly to `-run`, it's for the source text in the `//go:generate` statement. Better to keep it as it is!